java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture3/tasks/_2/Number2.java

```diff
@@ -0,0 +1,193 @@
+        account.firstName = scanner.next();
+        System.out.println("Type user last name:");
+        account.lastName = scanner.next();
+        System.out.println("Type your country");
+        System.out.println("Ukraine, USA, Canada, Mexico, Poland, Germany, France, Italy, Turkey");
+        account.country = scanner.next();
+        System.out.println("Type user email:");
+        account.userEmail = scanner.next();
+        System.out.println("Type user phone number:");
+        account.phoneNumber = scanner.next();
+        System.out.println("Type birth day");
+        account.birthDay = scanner.nextInt();
+        System.out.println("And birthday month");
+        account.birthMonth = scanner.nextInt();
+        validator.isValid(account);
+    }
+}
+
+interface Verification {
+    public boolean isValid(Account account);
+}
+
+class Account {
+    int userId;
+    String firstName;
+    String lastName;
+    String country;
+    String userEmail;
+    String phoneNumber;
+    int birthDay;
+    int birthMonth;
+}
+
+class Validator implements Verification {
+    userIdValidator userIdValidator = new userIdValidator();
+    nameValidation nameValidation = new nameValidation();
+    countryValidation countryValidation = new countryValidation();
+    userEmailValidator userEmailValidator = new userEmailValidator();
+    phoneNumberValidator phoneNumberValidator = new phoneNumberValidator();
+    birthDayAndBirthMonthValidator birthDayAndBirthMonthValidator = new birthDayAndBirthMonthValidator();
+
+    @Override
+    public boolean isValid(Account account) {
+        if (
+                userIdValidator.isValid(account)
+                        && nameValidation.isValid(account)
+                        && countryValidation.isValid(account)
+                        && userEmailValidator.isValid(account)
+                        && phoneNumberValidator.isValid(account)
+                        && birthDayAndBirthMonthValidator.isValid(account)
+        ) {
+            System.out.println("User id is: " + account.userId);
+            System.out.println("User name is: " + account.firstName + " " + account.lastName);
+            System.out.println("User country is: " + account.country);
+            System.out.println("User email is: " + account.userEmail);
+            System.out.println("User phone number is: " + account.phoneNumber);
+            System.out.println("User birthday is: " + account.birthDay + "/" + account.birthMonth);
+            return true;
+        }
+        return false;
+    }
+}
+
+class userIdValidator implements Verification {
+
+    @Override
+    public boolean isValid(Account account) {
+        if (account.userId > 0) {
+            System.out.println("User Id is Correct");
+            return true;
+        }
+        System.out.println("Invalid ID");
+        return false;
+    }
+}
+
+class nameValidation implements Verification {
+
+    @Override
+    public boolean isValid(Account account) {
```

Comment:
The task description contained a list of required validators, please carefully read it:

```
string is not empty, valid email, valid phone number, number is in the range between X and Y,
number is bigger than a value X, number is less than a value X
```